uplink/metainfo/client.go

```diff
@@ -408,6 +408,32 @@ func (client *Client) CommitObject(ctx context.Context, streamID storj.StreamID)
 	return Error.Wrap(err)
 }
 
+// GetObject gets single object
+func (client *Client) GetObject(ctx context.Context, bucket []byte, encryptedPath []byte, version int32) (_ storj.Object, _ storj.StreamID, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	response, err := client.client.GetObject(ctx, &pb.ObjectGetRequest{
+		Bucket:        bucket,
+		EncryptedPath: encryptedPath,
+		Version:       version,
+	})
+	if err != nil {
+		return storj.Object{}, storj.StreamID{}, Error.Wrap(err)
+	}
+
+	object := storj.Object{
+		Bucket: storj.Bucket{
+			Name: string(response.Object.Bucket),
+		},
+		Path:    string(response.Object.EncryptedPath),
```

Comment:
nit: typecast to `storj.Path` instead of `string`.

Reply:
done